shared/validation/languages/ur.js

```diff
@@ -0,0 +1,50 @@
+import tokenizeWords from 'talisman/tokenizers/words';
+
+// Minimum of words that qualify as a sentence.
+const MIN_WORDS = 1;
+
+// Maximum of words allowed per sentence to keep recordings in a manageable duration.
+const MAX_WORDS = 25;
```

Comment:
How long does it approximately take to read out 25 words aloud?

Reply:
8 to 10 seconds approx